pkg/ocicni/ocicni.go

```diff
@@ -239,20 +245,15 @@ func initCNI(exec cniinvoke.Exec, cacheDir, defaultNetName string, confDir strin
 		cacheDir:     cacheDir,
 	}
 
-	if exec == nil {
-		exec = &cniinvoke.DefaultExec{
-			RawExec:       &cniinvoke.RawExec{Stderr: os.Stderr},
-			PluginDecoder: cniversion.PluginDecoder{},
-		}
-	}
-
 	nsm, err := newNSManager()
 	if err != nil {
 		return nil, err
 	}
 	plugin.nsManager = nsm
 
-	plugin.syncNetworkConfig()
+	if err := plugin.syncNetworkConfig(); err != nil {
+		logrus.Errorf("CNI sync network config failed: %v", err)
```

Comment:
For consistency with some other errors, perhaps.

```suggestion
		logrus.Errorf("failed to sync CNI network config: %v", err)
```

Reply:
@saschagrunert, sorry my bad! I mistook this as an error return.

Reply:
That said, there are some log lines that diverge from this rules per:

```console
   kwilczynski@rocinante  Go/ocicni   master ✔  $ rg -s 'logrus.*\(\"[a-z]' -C 0 -g '!vendor'
pkg/ocicni/ocicni.go
497:			logrus.Errorf("error building CNI runtime config: %v", err)
506:				logrus.Errorf("error loading cached network config: %v", err)
507:				logrus.Warningf("falling back to loading from existing plugins on disk")
605:			logrus.Errorf("failed to read CNI cache file %s: %v", cacheFile, err)
617:			logrus.Errorf("failed to unmarshal CNI cache file %s: %v", cacheFile, err)
621:			logrus.Warningf("unknown CNI cache file %s kind %q", cacheFile, cachedInfo.Kind)
632:			logrus.Warningf("missing CNI cache file %s ifname %q or netname %q", cacheFile, cachedInfo.IfName, cachedInfo.NetName)
```

Fancy correcting these while at it? Since we have "cleanups" here.

Reply:
Yep, done :+1: